nni/tools/nnictl/nnictl_utils.py

```diff
@@ -222,9 +222,18 @@ def stop_experiment(args):
     if experiment_id_list:
         for experiment_id in experiment_id_list:
             print_normal('Stopping experiment %s' % experiment_id)
-            experiments_config = Experiments()
-            experiments_dict = experiments_config.get_all_experiments()
-            rest_pid = experiments_dict.get(experiment_id).get('pid')
+            # retry up to 10 times to get the experiment metadata
+            for i in range(1, 11):
+                experiments_dict = Experiments().get_all_experiments()
+                experiment_info = experiments_dict.get(experiment_id)
+                if experiment_info is None:
+                    print_warning('Get experiment {} metadata failed, {} time retry...'.format(experiment_id, i))
+                else:
+                    break
+            if experiment_info is None:
+                print_error('Experiment {} metadata getting failed, please manually check its status in `~/nni-experiments/.experiment`.')
```

Comment:
Suggest putting debugging info here so that we can do a diagnosis when similar issues appears on pipeline.

Reply:
print information in `.experiment`